apps/journeys-admin/src/components/Editor/Slider/Content/Goals/GoalsList/GoalsList.spec.tsx

```diff
@@ -68,65 +67,36 @@ describe('GoalsList', () => {
     })
   })
 
-  describe('mdUp', () => {
-    beforeEach(() =>
-      (useMediaQuery as jest.Mock).mockImplementation(() => true)
-    )
-
-    it('should render the information drawer on the right', () => {
-      const { getByTestId, getByRole } = render(<GoalsList goals={goals} />)
-      fireEvent.click(getByRole('button', { name: 'Learn More' }))
-      expect(getByTestId('GoalInformation').parentElement).toHaveClass(
-        'MuiDrawer-paperAnchorRight'
-      )
+  describe('GoalsList', () => {
+    it('should render title and subtitle', () => {
+      const { getByText } = render(<GoalsList goals={goals} />)
+      expect(getByText('The Journey Goals')).toBeInTheDocument()
     })
 
-    it('should close information drawer on close icon click', () => {
-      const { getByTestId, getByText, getByRole } = render(
-        <GoalsList goals={goals} />
-      )
+    it('should call dispatch with a URL when "Learn More" button is clicked', () => {
+      const { getByRole } = render(<GoalsList goals={goals} />)
       fireEvent.click(getByRole('button', { name: 'Learn More' }))
-      expect(getByText('Information')).toBeInTheDocument()
-      fireEvent.click(getByTestId('X2Icon'))
-      expect(getByTestId('GoalInformation').parentElement).not.toHaveClass(
-        'MuiDrawer-parentAnchorRight'
-      )
+      expect(useEditor().dispatch).toHaveBeenCalledWith({
```

Comment:
```suggestion
      expect(dispatch).toHaveBeenCalledWith({
```
Can we change this just to 'dispatch', in order to keep consistent with line 93?